src/Params/ReportMakeParams.php

```diff
@@ -0,0 +1,233 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Avtocod\B2BApi\Params;
+
+class ReportMakeParams
```

Comment:
```suggestion
final class ReportMakeParams
```

Reply:
To be honest, I don't really like the `final`. I prefer interfaces. Please, see realization here 4c3934bb9e61a2c195bfdcc4b92e9591ce37cc34